bee-message/bee-byte-cost-derive/src/builder.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use proc_macro2::TokenStream;
+use quote::{quote, ToTokens};
+use syn::Ident;
+
+pub(crate) struct ByteCostBuilder {
+    ident: Ident,
+    pub key: TokenStream,
```

Comment:
is it important that these are `pub` and not `pub(crate)` given that `ByteCostBuilder` is `pub(crate)`?